modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/RuntimeProvisioningException.java

```diff
@@ -16,6 +16,9 @@
 
 package org.wso2.carbon.appfactory.provisioning.runtime;
 
+/**
+ * Runtime provisioning API related logs
```

Comment:
Isn't this suppose to be "Runtime provisioning API related Exceptions" rather than logs ?